src/Compilers/CSharp/Portable/Compiler/Instrumentation.cs

```diff
@@ -199,11 +199,52 @@ public override BoundNode Visit(BoundNode node)
                         }
                         break;
                     case BoundKind.LocalDeclaration:
```

Comment:
if there is no initializer, there will be no code emitted corresponding to this bound node. I would think that only initialization case should be instrumented.